DEVELOPMENT.md

```diff
@@ -60,6 +60,17 @@ $ pytest --cov=cleanlab/ --cov-config .coveragerc --cov-report=html
 The coverage report will be available in `coverage_html_report/index.html`,
 which you can open with your web browser.
 
+
+### Type checking
+
+Cleanlab uses [mypy](https://mypy.readthedocs.io/en/stable/) typing. Type checking happens automatically during CI but can be tested locally.
```

Comment:
nit: "tested" seems like a slightly weird verb to use. Maybe substitute with "run".

Reply:
Learned some verb tense today! Initial instinct was to put "ran" 😅

Reply:
TIL

(learned this acronym from Anish xD)